docs/user/troubleshooting-guides/_sidebar.md

```diff
@@ -0,0 +1,16 @@
+* [Home](/)
```

Comment:
Except for the Home link, all the following links are broken.

Reply:
They will work when I add the `/api-gateway/` alias to `index.html` in `kyma`. I tested it on my local preview, and it seems to be working fine (like in this case: [sidebar](https://github.com/kyma-project/kyma/blob/main/docs/03-tutorials/00-api-exposure/_sidebar.md?plain=1), [website](https://kyma-project.io/#/03-tutorials/00-api-exposure/README)). I tried using relative links, but since the troubleshooting content is pulled from multiple folders, I'd have to add a separate README file and sidebar to each of them